plugins/migration/bin/rename.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A script that automatically upgrades pieces of the Blockly API
+ *     that have been renamed by modifying developers' local files.
+ */
+'use strict';
+
+import {createSubCommand, extractRequiredInfo} from './command.js';
+import fetch from 'node-fetch';
+import {readFileSync, writeFileSync} from 'fs';
+import * as versionUtils from './versions.js';
+import JSON5 from 'json5';
+
+
+const DATABASE_URL = `https://raw.githubusercontent.com/google/blockly/develop/scripts/migration/renamings.json5`;
+
+export const rename = createSubCommand(
+    'rename', '>=5', 'Perform renamings for breaking changes')
+    .option(
+        '-i, --in-place [suffix]',
+        'do renamings in-place, optionally create backup files with the ' +
+        'given suffix. Otherwise output to stdout')
+    .action(async function() {
+      const {fromVersion, toVersion, fileNames} = extractRequiredInfo(this);
+
+      const renamer = new Renamer(await getDatabase(), fromVersion, toVersion);
+      fileNames.forEach((name) => {
+        const contents = readFileSync(name, 'utf8');
+        const newContents = renamer.rename(contents);
+        const i = this.opts().i;
+        if (i) {
+          if (typeof i == 'string') writeFileSync(name + i, contents);
```

Comment:
I know we discussed allowing single-line if statements, but I thought the consensus was only for things like `if (a) return;` and not for actual statements that do things. I would strongly prefer this be on its own line, though we can revisit this as a team if opinions are split.